scripts/game.js

```diff
@@ -47,8 +48,10 @@ function preload() {
 function create() {
     createMario(this.physics, this.anims);
     createPlatforms(this.physics);
+    createPipes(this.physics);
 
     this.physics.add.collider(mario, platforms);
+    this.physics.add.collider(mario, pipes);
```

Comment:
In the original game, Mario does not collide with the pipes

Reply:
Didn't know this. Thought it worked as in the original Super Mario game where you could enter the pipes :). Anyway, fixed